guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,389 @@
+| *Address-based model*           | *Contact-based model* | *More intuitive and familiar*  |
+
+
+## Labels & contacts
+
+### Labels
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/adding-labels-to-static-address.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/adding-labels-to-static-address@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/silent-payments/adding-labels-to-static-address@2x.png"
+   alt-text = "An application screen showing labels being added to a static address."
+   caption = "Adding labels to a static address before posting or sharing with others."
+   width = 250
+   height = 542
+   layout = "float-right-desktop -background -shadow"
+%}
+
+Labels or tags (such as `business` or `no-kyc`) are brief identifiers often associated with information such as payments, transactions and addresses. They're used to organise, categorise or quickly spot items from a large set.
+
+Silent payments allow users to add labels to their static addresses on a case-by-case basis before sharing or posting them. When a payment is received, these labels can be detected by the receiver (but no one else) and used to identify the static address that was used to make the payment.
+
+</div>
+
+For example, Alice can add can customise her static address with different labels before sharing them
+- with a certain client
+- on Nostr and
+- on her website
+
+When she receives payments from any of these sources, the respective label(s) will be detected in the on-chain address and used to infer the source of the payment.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/silent-payments/how-silent-payment-labels-work.png"
+   retina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payment-labels-work@2x.png"
+   mobile = "/assets/images/guide/how-it-works/silent-payments/how-silent-payment-labels-work-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/silent-payments/how-silent-payment-labels-work-mobile@2x.png"
+   modalImage = "/assets/images/guide/how-it-works/silent-payments/how-silent-payment-labels-work@2x.png"
+   alt-text = "An image that illustrates how labels feature in silent payments works."
+   caption = "Alice customizes her static address with labels before she shares them. Whenever she receives a payment from one of them, the label is auto-applied and detected by the wallet."
+   width = 1024
+   height = 801
+   layout = "full-width"
+%}
+
+This scheme improves labelling in general since labels get auto-applied to all derived on-chain addresses. This eliminates the friction of manually adding labels to addresses or transactions. Labels can contain crucial information that is useful for automatic or manual coin selection for future outgoing payments.
```

Comment:
```suggestion
This scheme improves labelling in general, since labels get auto-applied to all derived on-chain addresses. This eliminates the friction of manually adding labels to addresses or transactions. Labels can contain crucial information that is useful for automatic or manual coin selection for future outgoing payments.
```